web_app.py

```
@@ -2915,84 +3088,133 @@ def worker() -> None:
+                    pending_ticket = None
+
+                if mind_thread is not None:
+                    mind_thread.join()
+                mind_payload = mind_box[0] if mind_box else None
                 if mind_payload:
                     ev_queue.put({
                         "type": "mindreading",
                         "payload": mind_payload,
                         "chat_turn_id": chat_turn_id,
                     })
                 # #544：流式不挡流——done 后补挂高亮（超时封顶）；有清单才发事件。
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)